classes/Category.php

```diff
@@ -2339,7 +2339,7 @@ public static function addToShop(array $categories, $idShop)
         $return = Db::getInstance()->execute($sql);
         // we have to update position for every new entries
         foreach ($tabCategories as $category) {
-            /* @var Category $category */
+            // @var Category $category
```

Comment:
I guess the original author meant 

```suggestion
            /**
             * @var Category $category
             */
```

Reply:
I had to move you suggestion outside the forearch statement to make the test pass.